app/src/main/java/org/mozilla/focus/Components.kt

```diff
@@ -205,6 +209,7 @@ private fun createCrashReporter(context: Context): CrashReporter {
     return CrashReporter(
         context = context,
         services = services,
+        telemetryServices = listOf(GleanCrashReporterService(context)),
```

Comment:
@Dexterp37 I'm going to sneak in this change as well to enable Glean crash pings as well. 🙂

Reply:
cool :-) (make sure to request a data review for adding this, too)

Reply:
Oh thanks, I didn't know that needed one too.

Reply:
Left a note in https://github.com/mozilla-mobile/focus-android/issues/4901 to make this a requirement.